bmlt-meeting-list.php

```diff
@@ -1208,10 +1208,25 @@ function bmlt_meeting_list($atts = null, $content = null) {
 						$data = utf8_encode($data);
 						$this->mpdf->WriteHTML($data);
 						$ph = intval($this->options['margin_bottom']) + intval($this->options['margin_top']) + $this->mpdf->y + -intval($this->options['page_height_fix']);
+      if ($this->options['margin_bottom'] == '5') {
+        $ph_footer_fix_bot = 0;
+      }
+      if ($this->options['margin_top'] == '5') {
+        $ph_footer_fix_top = 0;
+      }
+      if ($this->options['margin_bottom'] < '5') {
+        $ph_footer_fix_bot = 5 - $this->options['margin_bottom'];
+      }
+      if ($this->options['margin_top'] < '5') {
+        $ph_footer_fix_top = 5 - $this->options['top'];
+      }
+      $PH_FOOTER_MM_HEIGHT_ADJUST = 22;
+      $PH_FOOTER_MM = $ph_footer_mm_height_adjust + $ph_footer_fix_top + $ph_footer_fix_bot;
```

Comment:
need to reference the right variable name.  have you re-tested this after you made changes?

Reply:
I have not, im at airport with limited internet.